README.md

```diff
@@ -145,3 +165,311 @@ Releases are aligned to Vega core [releases](https://github.com/vegaprotocol/veg
+
+4. Once your computer has restarted, open the Microsoft Store by searching for "Microsoft Store" in the Start menu.
+
+5. In the Microsoft Store, search for the Linux distribution you want to install, such as Ubuntu, Debian, or OpenSUSE. Select the desired distribution from the search results.
+
+6. Click the **Install** button to download and install the Linux distribution.
+
+7. After the installation is complete, you can launch the Linux distribution from the Start menu or by searching for its name.
+
+8. The first time you launch the Linux distribution, it will initialize and set up your Linux environment. This might take a few minutes.
+
+9. Follow the on-screen prompts to create a username and password for your Linux environment.
+
+Once the setup is complete, you will have a functioning Windows Subsystem for Linux (WSL) installation on your Windows system. You can open a Linux terminal and start using Linux commands and tools within the WSL environment.
+  
+## 2. Install 'Make' to your WSL
+1. Open your terminal in Ubuntu.
+
+2. Update the package lists for upgrades and new package installations by running the following command (You may be asked for your password here):
+
+   ```
+   sudo apt update
+   ```
+
+3. Once the package lists are updated, you can install `make` by running the following command:
+
+   ```
+   sudo apt install make
+   ```
+
+4. The package manager will ask for your confirmation to install the package. Type 'Y' and press Enter to proceed with the installation.
+
+5. Ubuntu will download and install the `make` package along with any necessary dependencies.
+
+6. Once the installation is complete, you can verify that `make` is installed by checking its version:
+
+   ```
+   make --version
+   ```
+
+   This command will display the version of `make` installed on your Ubuntu system.
+
+## 3. Install 'Poetry' to your WSL
+To install Poetry on Windows Subsystem for Linux (WSL), you can follow these steps:
+
+1. Update the package lists and upgrade installed packages by running the following command:
+
+    ```
+    sudo apt update
+    ```
+
+2. Install the dependencies required for Poetry using the following command:
+
+   ```
+   sudo apt install python3-pip python3-venv
+   ```
+
+3. Once the dependencies are installed, you can proceed with the installation of Poetry. Use the following command to download and install Poetry:
+
+   ```
+   curl -sSL https://install.python-poetry.org | python3 -
+   ```
+
+4. After the installation is complete, you'll need to close and reopen your terminal or run the command `source $HOME/.poetry/env` to start using Poetry without restarting.
+
+5. Verify the installation by running the following command:
+
+   ```
+   poetry --version
+   ```
+
+   This command will display the version of Poetry installed on your system.
+
+## 4. Clone the Repository
```

Comment:
Again, I don't know if we want all those details? Maybe we do? @TomMcL ? I would just say "clone the repo".

Reply:
Updated.